python-checks/src/main/java/org/sonar/python/checks/regex/VerboseRegexCheck.java

```diff
@@ -19,16 +19,53 @@
  */
 package org.sonar.python.checks.regex;
 
+import java.util.List;
+import java.util.Optional;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import javax.annotation.Nullable;
 import org.sonar.check.Rule;
+import org.sonar.plugins.python.api.IssueLocation;
 import org.sonar.plugins.python.api.tree.CallExpression;
+import org.sonar.python.quickfix.IssueWithQuickFix;
+import org.sonar.python.quickfix.PythonQuickFix;
+import org.sonar.python.quickfix.PythonTextEdit;
+import org.sonar.python.regex.PythonRegexIssueLocation;
+import org.sonarsource.analyzer.commons.regex.RegexIssueLocation;
 import org.sonarsource.analyzer.commons.regex.RegexParseResult;
+import org.sonarsource.analyzer.commons.regex.ast.RegexSyntaxElement;
 import org.sonarsource.analyzer.commons.regex.finders.VerboseRegexFinder;
 
 @Rule(key = "S6353")
 public class VerboseRegexCheck extends AbstractRegexCheck {
 
+  private static final String ISSUE_MESSAGE_REGEX = ".+syntax '(.+)' instead of.+";
+  private static final Pattern issueMessagePattern = Pattern.compile(ISSUE_MESSAGE_REGEX);
+  public static final String QUICK_FIX_FORMAT = "Replace with \"%s\"";
+
   @Override
   public void checkRegex(RegexParseResult regexParseResult, CallExpression regexFunctionCall) {
     new VerboseRegexFinder(this::addIssue).visit(regexParseResult);
   }
+
+  @Override
+  public PreciseIssue addIssue(RegexSyntaxElement regexTree, String message, @Nullable Integer cost, List<RegexIssueLocation> secondaries) {
+    return Optional.of(super.addIssue(regexTree, message, cost, secondaries))
+      .map(IssueWithQuickFix.class::cast)
+      .map(issue -> {
+        Matcher matcher = issueMessagePattern.matcher(message);
```

Comment:
I'm not too fond of parsing an issue message. However, I don't have a better idea.
@guillaume-dequenne-sonarsource: food for thought.